pkgs/by-name/du/duckdb/update.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p cacert jq git moreutils nix nix-prefetch-github
+#!nix-shell -i bash -p cacert jq git moreutils nix nix-prefetch-github gh
```

Comment:
I noticed that the script calls `gh` but it is not defined as a dependency (I didn't had it on my machine)